include/nuttx/sched.h

```diff
@@ -160,7 +160,11 @@
 #  define _SCHED_ERRNO(r)            (-(r))
 #  define _SCHED_ERRVAL(r)           (r)
 #else
+#if defined(CONFIG_SCHED_THREAD_LOCAL)
+#  define _SCHED_GETTID()            sched_gettid()
```

Comment:
remove _SCHED_GETTID since we don't have the different implementation between kernel and userspace.